uint128/uint128.go

```diff
@@ -0,0 +1,121 @@
+package uint128
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+type Int [2]uint64
+
+func SetUint64(lo uint64) *Int {
```

Comment:
This should be a receiver, e.g., `func (i *Int) SetUint64(lo uint64) *Int` like in `math/big` and `core/felt`.